deltachat-ios/Helper/ShortcutManager.swift

```diff
@@ -246,7 +246,7 @@ public class ShortcutManager {
                  img.src = "data:image/png;base64,\(addToHomeIconBase64)";
                  img.classList.add("iconImage");
                  div.appendChild(img);
-                 var node = document.createTextNode("\(String.localized("shortcut_add_to_home_description"))");
+                 var node = document.createTextNode("\(String.localized("shortcut_step2_tap_add_to_home_screen").replacingOccurrences(of: "\"", with: " - "))");
```

Comment:
we probably want this hack also for `shortcut_add_to_home_description` that may also get quotes easily in some translations (`"Teilen" drücken`)